components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gene_suggestions.xml

```diff
@@ -121,16 +121,67 @@
 $xwiki.jsx.use('PhenoTips.PanelsExport')##
 $xwiki.jsx.use('PhenoTips.PanelsDisplay')##
 $xwiki.ssx.use('PhenoTips.PanelsDisplay')##
+$xwiki.jsfx.use('js/xwiki/table/livetable.js', true)##
+$xwiki.ssfx.use('js/xwiki/table/livetable.css', true)##
   (% class="instant-search gene-panels-search-box" %)(((
-    === $services.localization.render('phenotips.UIXField.geneSuggestions.geneSearch.matchingGenes')  {{html wiki=false clean=false}}&lt;span class="buttonwrapper"&gt;&lt;a href="#" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" id="genePanelsExport" class="button gene-panels-button csv-export-button"&gt;&lt;span class="fa fa-fw fa-download"&gt;&lt;/span&gt; $services.localization.render('phenotips.UIXField.geneSuggestions.download')&lt;/a&gt;&lt;/span&gt;{{/html}}{{icon name="question-circle" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" cssClass="xHelpButton"/}} ===
+    === $escapetool.xml($services.localization.render('phenotips.UIXField.geneSuggestions.geneSearch.matchingGenes'))  {{html wiki=false clean=false}}&lt;span class="buttonwrapper"&gt;&lt;a href="#" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" id="genePanelsExport" class="button gene-panels-button csv-export-button"&gt;&lt;span class="fa fa-fw fa-download"&gt;&lt;/span&gt; $services.localization.render('phenotips.UIXField.geneSuggestions.download')&lt;/a&gt;&lt;/span&gt;{{/html}}{{icon name="question-circle" title="$services.localization.render('phenotips.UIXField.geneSuggestions.download.hint')" cssClass="xHelpButton"/}} ===
 
     (% class="gene-search-box-content"%)(((
-      (% class="hint"%)$services.localization.render('phenotips.UIXField.geneSuggestions.geneSearch.Hint')(%%)##
+      (% class="hint"%)$escapetool.xml($services.localization.render('phenotips.UIXField.geneSuggestions.geneSearch.Hint'))(%%)##
       {{html wiki=false clean=false}}&lt;input type="hidden" id="panelsSearchTermsInput" value="" /&gt;&lt;input type="hidden" id="panelsSearchTermIDsInput" value="" /&gt;{{/html}}##
       {{html wiki=false clean=false}}&lt;input type="hidden" id="panels-phenotypes-summary" value="$escapetool.xml($jsontool.serialize($phenotypesSummary))" /&gt;{{/html}}##
       (% id="panelsSearchTerms" %)((()))
       (% class="background-search gene-search hidden" %)(((
-        (% id="gene-search-results" class="background-suggestions" %)((()))
+        (% id="gene-search-results" class="background-suggestions" %)(((
+          {{html wiki=false clean=false}}
+            &lt;table id="panels-livetable" class="xwiki-livetable"&gt;
```

Comment:
What prevents you from using the `#livetable` macro which normally generates all this HTML?